modules/auth.js

```diff
@@ -65,35 +47,31 @@ check = function (req, res, next) {
 
 allow = function (rights) {
   return function (req, res, next) {
-    var cip = req.session[cas.session_name];
-    User.findOne({ cip: cip }, function (err, obj) {
-      var found = false;
-      if (obj) {
-        rights.forEach(function (element, index, array) {
-          if (obj.rights == element) {
-            found = true;
-            next();
-          }
-        });
-      }
-      
-      if (!found) {
-        res.status('403');
-        res.send('403: Forbidden');
-      }
+    userRights = req.session.userInfo.rights;
+    var userAllowed = false
+    userRights.forEach(function(userRight) {
+      rights.forEach(function(right) {
+        if (userRight === right) {
+          next();
+          userAllowed = true;
+        }
+      })
     });
+    if (!userAllowed) {
+      res.redirect('/');
+    }
```

Comment:
I agree, we should have a standard HTTP 401 page.
(And 403 and 404 pages as well...)